recipes/pycairo/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "pycairo" %}
+{% set version = "1.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: git://git.cairographics.org/git/pycairo
+  git_tag: 5a45b0bfeccd3a0f2a6467bafcad515428439190
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - xorg-libx11
+    - python >=3
+    - cairo >=1.10.2
+  run:
+    - xorg-libx11
+    - python >=3
+    - cairo >=1.10.2
+
+test:
+  imports:
+    - cairo
```

Comment:
Add this to the test section so that we have a record of what system libs are being linked against.
```yaml
  commands:
    - conda inspect linkages -p $PREFIX $PKG_NAME  # [linux]
    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```